src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -5,32 +5,35 @@ title: New Relic Azure Monitor integration
-- Coverage: New metrics will be available the moment Azure adds it to their Azure Monitor API endpoints, including data from new Azure services.
-- Always up-to-date: Any improvement in existing metrics will show up in New Relic automatically.
-- Faster metrics: The new integration enables customers to define up to 1 polling interval reducing the average time-to-ingest for most Azure Monitor namespaces.
+Our Azure Monitor integration improves upon our past Azure solutions: 
+
+* Coverage: New metrics will be available the moment Azure adds it to their Azure Monitor API endpoints, including data from new Azure services.
+* Always up-to-date: Any improvement in existing metrics will show up in New Relic automatically.
+* Faster metrics: The new integration enables customers to define up to 1 polling interval reducing the average time-to-ingest for most Azure Monitor namespaces.
+
+## The Azure Native New Relic Service [#azure-native]
 
-## Features
+Our Azure Monitor integration is the integration that powers the Azure Native New Relic Service. This service lets you set up and configure New Relic directly from Azure, and store your New Relic data in Azure. [Learn more about the Azure Native New Relic Service.](/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native/)
 
-Monitor all your Azure metrics from all supported Azure services with our Azure Monitor integration.
+## Features [#features]
 
-Once you enable the New Relic Azure Monitor integration, New Relic queries your Azure platform services according to the configured polling interval. This enables all of the following features:
+Our Azure Monitor integration monitors all your Azure metrics from all supported Azure services. Once you enable it, we query your Azure platform services according to the configured polling interval. This enables all of the following features:
 
-- Metric ingestion from [all Azure services supported by Azure Monitor](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported)
-- Native [dimensional metrics](/docs/data-apis/understand-data/new-relic-data-types/#metrics-new-relic) experience for queries, alerts and data explorer.
-- Azure [entities](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/) in the [New Relic explorer](/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts/) with golden metrics and built-in dashboards.
-- Metrics decorated with custom tags defined in Azure resources
-- Faster [polling intervals](/docs/infrastructure/microsoft-azure-integrations/getting-started/polling-intervals-azure-integrations/) available (up to `1min`), enabling faster time-to-glass for metrics
-- Ability to filter monitored resources by group, type and tags
+* Metric ingestion from [all Azure services supported by Azure Monitor](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported).
```

Comment:
This is kind of passive. How about something more active like:

"* Ingest all the metrics from your Azure Monitor-supported services" or something?

Reply:
Look for opportunities to make the rest of the bullets more active.